wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsDFUClient.java

```diff
@@ -120,10 +120,25 @@ public WsDfuServiceSoapProxy getSoapProxy() throws Exception
      * @throws Exception
      */
     public DFUInfoResponse getFileInfo(String logicalname, String clustername) throws Exception
+    {
+      return this.getFileInfo(logicalname,  clustername, false);
+    }
+    /**
+     *
+     * @param logicalname logical file name, can start with ~
+     * @param clustername optional, if specified the cluster name used in the search
+     * @param wantRecordInfo want record structure information returned
+     * @return
+     * @throws Exception
+     */
+    public DFUInfoResponse getFileInfo(String logicalname, String clustername,
```

Comment:
let's rename this flag to fetchRecordInfo.
In fact, let's expose both flags: fetchBinaryRecordInfo, and fetchJSONRecordInfo.
It seems unnecessary to request the record info in 2 formats.

line 124 could change to return this.getFileInfo(logicalname,  clustername, false, false)